### PR TITLE
Change to new order format & clean up orders

### PR DESCRIPTION
**Pull request by martijnjanssen**
* Change to new order response
* Encapsulate some js
* Simplify methods & remove redundant functions
* Change bindings to one-way bindings

### CODE REVIEW

File: src/lancie-my-area/my-area-order-item.html [hunk 1]
```diff
@@ -99,22 +99,18 @@
...
     },
-    _format: function(money) {
-      var n = money,
-        c = 2,
-        d = ',',
-        t = '.',
-        s = n < 0 ? '-' : '',
-        i = parseInt(n = Math.abs(+n || 0).toFixed(c)) + '',
-        j = (j = i.length) > 3 ? j % 3 : 0;
-      return s + (j ? i.substr(0, j) + t : '') + i.substr(j).replace(/(\d{3})(?=\d)/g, '$1' + t) + (c ? d + Math.abs(n - i).toFixed(c).slice(2) : '');
```
Comment (MatthijsKok) [inline src/lancie-my-area/my-area-order-item.html]: nice job removing this monstrosity 👍

File: src/lancie-my-area/my-area-orders/lancie-ticket-item.html [hunk 2]
```diff
@@ -64,37 +57,28 @@
           value: '',
           notify: true
         },
-        options: String,
-        price: String,
-        accStatus: String
       },
+
       updateTicketInfo: function() {
```
Comment (TimvdLippe) [inline src/lancie-my-area/my-area-orders/lancie-ticket-item.html]: This function can be removed as a whole I suppose?

Comment (TimvdLippe) [inline src/lancie-my-area/my-area-orders/lancie-ticket-item.html]: Then the observer on the ticket property should also be removed

File: src/lancie-my-area/my-area-orders/lancie-ticket-item.html [hunk 3]
```diff
@@ -64,37 +57,28 @@
...
-          return 'Pickup Service';
-        } else if (this.ticket.chMember) {
-          return 'Christiaan Huygens Member';
+
+      _getOptions: function(options) {
+        if (options.length <= 0) {
+          return 'No options selected';
         }
-        return 'No extra options';
+        var optionString =  options.reduce(function(a, b) {
```
Comment (TimvdLippe) [inline src/lancie-my-area/my-area-orders/lancie-ticket-item.html]: Concepts of programming languages was useful :tada:

**Comment by MatthijsKok** [comment]
I'm function-testing atm

File: src/lancie-my-area/my-area-orders/lancie-ticket-item.html (hunk 3 above)
Comment (martijnjanssen) [inline src/lancie-my-area/my-area-orders/lancie-ticket-item.html]: That, and big data processing :+1:

**Pull request merged**